plugins/nordic-hid/README.md

```diff
@@ -4,25 +4,27 @@ title: Plugin: Nordic HID
 
 ## Introduction
 
-This plugin is able flash the firmware for the hardware supported by `nRF52-Desktop`.
+This plugin is able to update the firmware for the hardware supported by [nRF Desktop](https://www.nordicsemi.com/Products/Reference-designs/nRF-Desktop) application reference design developed as part of the [nRF Connect SDK](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/applications/nrf_desktop/README.html).
 Tested with the following devices:
 
-* [nrf52840dk development kit](https://www.nordicsemi.com/Products/nRF52840)
-* [nRF52840 Dongle](https://www.nordicsemi.com/Products/Development-hardware/nrf52840-dongle)
-* nRF52840 Gaming Mouse
-* nRF52832 Desktop Keyboard
+* [nRF52840 Development Kit](https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/boards/nordic/nrf52840dk/doc/index.html)
+* [nRF52840 Dongle](https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/boards/nordic/nrf52840dongle/doc/index.html)
+* Boards specific to the nRF Desktop reference design:
+  * nRF52840 Gaming Mouse
+  * nRF52832 Desktop Keyboard
```

Comment:
```suggestion
* Boards specific to the nRF Desktop reference design:

  * nRF52840 Gaming Mouse
  * nRF52832 Desktop Keyboard
```

Reply:
The empty line is not needed for Markdown (see https://blog.markdowntools.com/posts/using-nested-lists-and-sub-bullets-in-markdown for an example)